plugin.go

```diff
@@ -2,31 +2,32 @@
 package plasmactlbump
 
 import (
-	"github.com/spf13/cobra"
-
 	"github.com/launchrctl/keyring"
 	"github.com/launchrctl/launchr"
+	"github.com/spf13/cobra"
```

Comment:
- [ ] Throw an error during propagation when a variable which exists in artifact but does not exist/never existted in repo
      (non existing value in group_vars in artifact)

      sed -i 's/account_port: .*/account_port: 8099/g' .compose/comparison-artifact/platform/group_vars/platform/vars.yaml 
      compose and sync

-> No message whereas error should be thrown